CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.1.0]
+### Uncategorized
+- Request validation should not throw if verifyingContract is not defined in typed signature ([#328](https://github.com/MetaMask/eth-json-rpc-middleware/pull/328))
+- Add changelog entries for `#318` ([#327](https://github.com/MetaMask/eth-json-rpc-middleware/pull/327))
+- remove eth_sign ([#320](https://github.com/MetaMask/eth-json-rpc-middleware/pull/320))
```

Comment:
This is a breaking change. Can we create a back port v14.0.1 release instead with just the bug fix? So that we can omit the breaking change and simplify the process of shipping this.

Reply:
I have created a `14.x` branch that we can use as the target for a v14.0.1 RC.

Feel free to create the RC manually, it will need to be published manually anyway.